mob version

AlinaWhite opened the pull request:
all done just without JS

AlinaWhite committed:
mobil version

all done just without JS

AlinaWhite committed:
ipad css

додалп стилі на планшет але ще додам фонове зображення зараз коли буду вже на ноут робити

BulatovaTati reviewed (changes requested):
1.https://www.awesomescreenshot.com/video/33827019?key=cece432a106b0b6f9c45f3fd9f7fa447
2. https://www.awesomescreenshot.com/video/33827771?key=bd4abbc283e384907a1c7d1543190341

BulatovaTati reviewed (changes requested):
Зміни не внесені.

AlinaWhite committed:
all css together

я не буду мати часу завтра нажалт скидати окремо стилі на планшет і ноут  скидаю все разом щоб вже залишився JS and backgraung img на завтра якщо тут все добре
дякую
